tests/page-objects/default/contacts/contacts.wdio.page.js

```diff
@@ -4,7 +4,7 @@ const genericForm = require('../enketo/generic-form.wdio.page');
 const commonElements = require('../common/common.wdio.page');
 const sentinelUtils = require('../../../utils/sentinel');
 const modalPage = require('../../../page-objects/default/common/modal.wdio.page');
-const searchBox = () => $('.mm-search-bar-container input#freetext');
+const searchBox = () => $('input#freetext');
```

Comment:
@lorerod @ngaruko  I think more specific selectors are better, why the previous one didn't work?